integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/data/value/ValueConverter.java

```diff
@@ -13,7 +13,8 @@ public interface ValueConverter {
   CertificateQuestionValueText NOT_PROVIDED_VALUE = CertificateQuestionValueText.builder()
       .value(NOT_PROVIDED)
       .build();
-
+  String MISSING_LABEL = "Saknas";
```

Comment:
This value is displayed when wrong config is used if I understand the code correct. Im leaning more towards that we display a TECHNICAL_ERROR when these unexpected events occur.

Maybe I have misunderstood the usage of TECHNICAL_ERROR? Lets discuss it if you disagree!

Reply:
MISSING_LABEL is being used in this context for other values with headings, so I'm assuming it's been discussed last week